tests/test-mongoc-collection.c

```diff
@@ -1243,7 +1243,8 @@ test_command_fq (void)
    client = mongoc_client_new (gTestUri);
    ASSERT (client);
 
-   collection = get_test_collection (client, "$cmd.sys.inprog");
+   collection = mongoc_client_get_collection (client,
```

Comment:
@alexeyvo , you think this is an error this guys have in the library itself? Nothing wrong with the database version 3 then?